tools/ResourceMetaSchema.json

```diff
@@ -195,8 +195,7 @@
             }
           },
           "required": [
-            "oneOf",
```

Comment:
Why is this part of this checkin? Is ARM OK with this? Having a description is probably what they want?